test/models/multiple-results-event.tests.js

```diff
@@ -0,0 +1,427 @@
+      input.resultSetEndTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is object', () => {
+      input.resultSetEndTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+  });
+
+  describe('format', () => {
+    let input;
+    const results = ['Invalid', '1', 'y'];
+
+    beforeEach(() => {
+      input = {
+        txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        txStatus: TX_STATUS.SUCCESS,
+        blockNum: 5,
+        address: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        version: 6,
+        name: "Test",
+        numOfResults: 2,
+        centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        betStartTime: 1560965704,
+        betEndTime: 1560965704,
+        resultSetStartTime: 1560965704,
+        resultSetEndTime: 1560965704,
+        escrowAmount: "1000000",
+        arbitrationLength: 172800,
+        thresholdPercentIncrease: "10",
+        arbitrationRewardPercentage: 10,
+        consensusThreshold: "100000000000",
+        arbitrationEndTime: 1560965704,
+      };
+
+      input.results = filter(
+        map(results, (item) => {
+          try {
+            return web3.utils.utf8ToHex(item);
+          } catch (err) {
+            return '(parse error)';
+          }
+        }),
+        item => !!item,
+      );
+    });
+
+    it('It should pass if passing all inputs correctly', () => {
+      const event = new MultipleResultsEvent(input);
+      assert.equal(event.txid, input.txid);
+      assert.equal(event.txStatus, input.txStatus);
+      assert.equal(event.blockNum, input.blockNum);
+      assert.equal(event.address, input.address);
+      assert.equal(event.ownerAddress, input.ownerAddress);
+      assert.equal(event.version, input.version);
+      assert.equal(event.name, input.name);
+      assert.deepEqual(event.results, results);
+      assert.equal(event.numOfResults, input.numOfResults);
+      assert.equal(event.centralizedOracle, input.centralizedOracle);
+      assert.equal(event.betStartTime, input.betStartTime);
+      assert.equal(event.betEndTime, input.betEndTime);
+      assert.equal(event.resultSetStartTime, input.resultSetStartTime);
+      assert.equal(event.resultSetEndTime, input.resultSetEndTime);
+      assert.equal(event.escrowAmount, input.escrowAmount);
+      assert.equal(event.arbitrationLength, input.arbitrationLength);
+      assert.equal(event.thresholdPercentIncrease, input.thresholdPercentIncrease);
+      assert.equal(event.arbitrationRewardPercentage, input.arbitrationRewardPercentage);
+      assert.isString(event.consensusThreshold);
+      assert.isNumber(event.arbitrationEndTime);
+    });
+
+    it('It should pass if no betStartTime and resultSetEndTime', () => {
```

Comment:
create another `mutationInput` and move this test as part of the test above `It should pass if passing all inputs correctly`